catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -97,7 +98,7 @@
 import spark.Request;
 import spark.Response;
 
-public class EndpointUtil {
+public class EndpointUtil implements EndpointUtility {
```

Comment:
✏️ If we split this class up, then the `EndpointUtility` interface is no longer needed. And `EndpointUtil` need not implement anything.